src/stk/molecular/topology_graphs/topology_graph/construction_state/molecule_state/deletions_summary.py

```diff
@@ -57,6 +59,9 @@ def __init__(
         deleted_ids : :class:`iterable` of :class:`int`
             The ids of `atoms`, which should be deleted.
 
+        deleted_bond_ids : :class:`iterable` of :class:`tuple`
+            Tuple of atom ids of bonds that should be deleted.
```

Comment:
Might be worth creating a namedtuple and the using it here, so that the shape of the iterable is easy to understand.


(The docstring doesn't include the type because the code has a type hint.)


```python
from typing import NamedTuple

class _BondId(NamedTuple):
    """
    Identifies a bond in a molecule.

    Attributes
    ----------
    atom1_id
        The id of the first atom in the bond.
    
    atom2_id 
        The id of the sceond atom in the bond.

    """

    atom1_id: int
    atom2_id: int

```

```
    deleted_bond_ids : :class:`iterable` of :class:`._BondId`
        Ids of bonds that should be deleted
```

Reply:
Done. Added it to reactions_summary.py and updated docs.